DisguiseUnityRenderStream/Runtime/PipelineAbstraction/Presenter.cs

```diff
@@ -126,6 +126,15 @@ public bool clearScreen
         public Vector2 sourceSize => new Vector2(m_source.width, m_source.height);
         
         public Vector2 targetSize => new Vector2(Screen.width, Screen.height);
+        
+        Backend CreateBackend()
+        {
+#if UNITY_EDITOR
+            return m_backEnd = new EditorBackend(this);
```

Comment:
Wait a minute. How exactly are we running the project in the Editor?

Reply:
Right now it's only for devs, it's been really helpful for iterating on the pipeline abstraction without leaving the editor.